install_config/install/prerequisites.adoc

```diff
@@ -669,7 +669,9 @@ connections, and is only required if you have clustered etcd.
 * In the above examples, port *4789* is used for User Datagram Protocol (UDP).
 * When deployments are using the SDN, the pod network is accessed via a service proxy, unless it is accessing the registry from the same node the registry is deployed on.
 * {product-title} internal DNS cannot be received over SDN. Depending on the detected values of `*openshift_facts*`, or if the `*openshift_ip*` and `*openshift_public_ip*` values are overridden, it will be the computed value of `*openshift_ip*`. For non-cloud deployments, this will default to the IP address associated with the default route on the master host. For cloud deployments, it will default to the IP address associated with the first internal interface as defined by the cloud metadata.
-* The master host uses port *10250* to reach the nodes and does not go over SDN. It depends on the target host of the deployment and uses the computed values of `*openshift_hostname*` and `*openshift_public_hostname*`.
+* The master host uses port *10250* to reach the nodes and does not go over SDN.
+It depends on the target host of the deployment and uses the computed values of
+`*openshift_public_hostname*`.
```

Comment:
I think that neither of those variables are relevant to this particular line. The only thing that matters now is the host's hostname which is what it will use to register to the API.
@sjenning is this correct

Reply:
I believe that `openshift_public_hostname` is only used when generating certs (allowing a different external name to be included in the cert), not configuring the node.

Reply:
It's going to reach the kubelet via the hostname that the node registers with afaik. I'd just get rid of this last bullet point.